src/js/requestHandlers.js

```diff
@@ -69,15 +69,17 @@ fluid.defaults("sjrk.storyTelling.server.saveStoryWithBinariesHandler", {
 });
 
 sjrk.storyTelling.server.handleSaveStoryWithBinaries = function (request) {
+    var storyModel = JSON.parse(request.req.body.model);
 
-    // The list of uploaded files is available at request.req.files
-    console.log(request.req.files);
+    fluid.each(storyModel.content, function (block) {
+        if (block.blockType === "image") {
+            var imageFile = fluid.find_if(request.req.files.file, function (singleFile) {
+                return singleFile.originalname === block.fileDetails.name;
```

Comment:
We should use `singleFile.filename` here rather than `singleFile.originalname`, so that if our upload method renames the filename (it currently doesn't) this continues to work.